topi/python/topi/x86/conv2d.py

```diff
@@ -215,7 +215,8 @@ def default_schedule(op):
         s[C].reorder(fused, rc, h, wo, ry, rx, wi)  # move rc to outer loop
         s[C].unroll(rx)
         s[C].unroll(ry)
-        s[C].vectorize(wi)
+        if w.dom.extent.value % 16 == 0:
```

Comment:
Sorry, I've just sent a patch #1504 to remove this schedule completely. Can you undo this?